Homework_1/Task_1.py

```diff
@@ -0,0 +1,13 @@
+def longest_words(file_path):
+    with open(file_path, 'r', encoding='utf-8') as t:
+        text = t.read().split()
+        text_sorted = sorted(text, key=len)
+        max_len = len(max(text_sorted, key=len))
```

Comment:
Интересное решение с сортировкой, но не очень понятно, зачем его нужно делать тут. 
sorted по-умолчанию сортирует в порядке возрастания (или как часто любят писать в порядке неубывания).
Следовательно в отсортированном списке мы гарантированно знаем, что самое большое значение будет в конце. Тогда можно
```suggestion
        max_len = len(text_sorted[-1])
```

В твоем решении сортировка просто выходит лишним шагом, потому как дальше ты все равно проходишься по списку и сравниваешь длину каждой строки с максимум.